docs/running.md

```diff
@@ -11,28 +11,28 @@ Other than subtle differences described here,
 both approaches are equivalent as they can define
 the same parameters which will be processed via the same routines.
 
-!!! Note
-    Most examples in these documentation pages are in a
-    **Python API** format rather than a
-    **<a href="https://yaml.org/" target="_blank">YAML</a>
+> **Note:**
+    Most examples in these documentation pages are in a **Python API** format rather than a [yaml](https://yaml.org/)
```

Comment:
missing ** before [yaml]
```suggestion
    Most examples in these documentation pages are in a **Python API** format rather than a **[yaml](https://yaml.org/)
```